webpack.config.ts

```diff
@@ -64,7 +67,6 @@ const config: Configuration = {
 				from: '*.+(html|png)',
 			}],
 		}),
-		new SizePlugin({writeFile: false}),
```

Comment:
It's been 2 years and a half, it's time to move on.

### Before

```
        background.js ⏤  24.2 kB
          options.css ⏤  2.62 kB
         options.html ⏤  2.12 kB
           options.js ⏤  42.6 kB
   refined-github.css ⏤  16.9 kB
    refined-github.js ⏤  152 kB
 resolve-conflicts.js ⏤  1.33 kB
```

### After

```
Entrypoint refined-github 690 KiB = refined-github.css 55.5 KiB refined-github.js 635 KiB
Entrypoint background 121 KiB = background.js
Entrypoint options 187 KiB = options.css 6.86 KiB options.js 180 KiB
Entrypoint resolve-conflicts 3.5 KiB = resolve-conflicts.js
webpack compiled successfully in 8107 ms
```

Good enough. Plus it shows the real size instead of the misleading gzipped size